docs/the-algorithm.rst

```diff
@@ -0,0 +1,132 @@
+
+    The four points in our box, with their connections.
+
+This leads to a confusion situation, where the "correct" interpretation of our antimeridian-crossing box would be globe-covering shape with a "hole" over the equator and the prime meridian.
+This is very rarely what you're actually trying to represent.
+
+.. figure:: img/winding-repercussions.png
+    :scale: 30%
+    :align: center
+
+    If we interpret our antimeridian-crossing box literally, it describes a very strange geometry.
+
+Here's where our algorithm comes in.
+
+Segmentation
+~~~~~~~~~~~~
+
+Given our antimeridian-crossing box as our input, we walk through the points until we find a longitude jump of more than 180° degrees.
+
+.. figure:: img/delta-gt-180.png
+    :scale: 30%
+    :align: center
+
+    A longitude jump greater than 180°.
+
+When we find such a jump, we split the polygon into segments by inserting two points, one on each antimeridian.
+
+.. figure:: img/segment.png
+    :scale: 30%
+    :align: center
+
+    Creating two segments by adding points on the antimeridian.
+
+We then continue walking through the points, applying the same procedure when other jumps of 180° longitude are found.
+
+.. figure:: img/another-delta.png
+    :scale: 30%
+    :align: center
+
+    Another jump of more than 180° longitude.
+
+When we finish walking the points, we join the last segment back to the first.
+
+.. figure:: img/finish-segmentation.png
+    :scale: 30%
+    :align: center
+
+    Finishing the segmentation step.
+
+Finally, segment end are joined to segment starts by searching up or down the antimeridian to create multiple polygons.
+This conforms to the `GeoJSON specification <https://datatracker.ietf.org/doc/html/rfc7946#section-3.1.9>`_, which expects antimeridian-crossing polygons to be split into multipolygons at  the antimeridian.
+
+.. figure:: img/polygons.png
+    :scale: 30%
+    :align: center
+
+    The output polygons.
+
+The poles
+~~~~~~~~~
+
+Geometries that overlap the north or the south pole present a related, but different, problem.
+Here's a oval geometry that covers the north pole, as visualized from "above" the earth:
+
+.. figure:: img/pole.png
+    :scale: 30%
+    :align: center
+
+    A polygon encircling the north pole.
+
+That geometry would be represented as a set of points, all with positive latitudes and with longitudes covering more-or-less the entire -180° to 180° extent of our coordinate system.
+Because the edge of our geometry does not touch the pole, there's no point that has a 90° latitude.
+This means that a classic cartesian visualization of the geometry never touches the north pole, since the maximum latitude of the geometry is less than 90°.
+
+.. figure:: img/pole-as-points.png
+    :scale: 30%
+    :align: center
+
+    The same polygon as points, visualized in a cartesian plot.
+
+Our algorithm includes a step after segmentation that "extends" segments over the pole, when appropriate, by adding two new points.
```

Comment:
I broke it down step-by-step, lmk what you think. I _didn't_ talk about `force_north_pole` because most folks won't know about it, so (hopefully) they'll (correctly) assume they get this out-of-the-box?